charm/metadata.yaml

```diff
@@ -0,0 +1,21 @@
+name: testflinger-k8s
+display-name: testflinger-k8s
+summary: Juju k8s charm for testflinger server
+
+description: |
+  Juju k8s charm for testflinger server
+
+containers:
+  testflinger:
+    resource: testflinger-image
+
+resources:
+  testflinger-image:
+    type: oci-image
+    description: OCI image for the 'testflinger' container
+    upstream-source: plars/testflinger:latest
```

Comment:
Is there any canonical testflinger image? Or are we okay with using yours?

Reply:
Not at this point, but it probably won't matter as there doesn't seem to be a way that I've found to force it to pull unless you specify a new tag every time. It always wants to use a cached copy. However, this doesn't seem to be the way to get the image anyway. When you upload the charm to the charm store, you also have to manually upload any resources (including the image) or else it won't work. So it won't actually get it from here anyway.

Also, from the juju documentation it tells us that this field is ignored by juju. It's just for reference.